grbl/grbl.h

```diff
@@ -41,10 +41,10 @@
 // Define the Grbl system include files. NOTE: Do not alter organization.
 #include "config.h"
 #include "nuts_bolts.h"
+#include "cpu_map.h"
```

Comment:
I needed to move this up so that `cpu_map.h` got processed before `settings.h` so that `settings.h` could know if `HAS_DIGIPOTS` was defined